doc/ref_impl/cntt-ri2/chapters/chapter02.md

```diff
@@ -22,10 +22,20 @@ interpreted as described in [RFC2119](https://www.ietf.org/rfc/rfc2119.txt).
 
 ## 2.2 Reference Architecture Specification
 
-| RA2 Section | RA2 Reference  | Description | Requirement for Basic Profile | Requirement for Network Intensive Profile| RI2 Traceability |
-|---|---|---|---|---|---|
-|Kubernetes Nodes|`ra2.ch.001`|Huge Pages|Not required|It must be possible to enable Huge Pages (2048KiB and 1048576KiB) within the Kubernetes Node OS, exposing schedulable resources `hugepages-2Mi` and `hugepages-1Gi`.|_TBC_|
-|||||||
+| RA2 Section | RA2 Reference  | Specification | Requirement | Requirement for Basic Profile | Requirement for Network Intensive Profile | RI2 Traceability |
```

Comment:
Could the values under RA2 Section list the actual RA2 Section # as in "4.2 Kubernetes Nodes" and not just "Kubernetes Nodes"?